lillian_klasen/login_reg/server.py

```diff
@@ -0,0 +1,96 @@
+from flask import Flask, render_template, request, redirect, flash, session
+from mysqlconnection import MySQLConnector
+import re
+import md5
+
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+
+app = Flask(__name__)
+app.secret_key = "secret"
+mysql = MySQLConnector(app, 'logins')
+
+@app.route('/')
+def index():
+  return render_template('index.html')
+
+
+@app.route('/process', methods=['POST'])
+def process():
```

Comment:
Let's talk about your validations. Would all of your validations get run for each of your inputs, or only one? What I mean by this is you may want to change all your elif statements to if statements so that all the validations get run no matter what.